test/integration/targets/aws_secret_facts/tasks/main.yaml

```diff
@@ -0,0 +1,90 @@
+---
+- block:
+  - name: set connection information for all tasks
+    set_fact:
+      aws_connection_info: &aws_connection_info
+        aws_access_key: "{{ aws_access_key }}"
+        aws_secret_key: "{{ aws_secret_key }}"
+        region: "{{ aws_region }}"
+        #security_token: "{{ security_token }}"
```

Comment:
CI uses temporary credentials so this needs to be uncommented. You can use `security_token: "{{ security_token | default() }}"` or `security_token: "{{ security_token | omit() }}"` which should let things pass if you don't provide a security token.

Reply:
Oh good point, I added `omit()`.  This was indeed messing with my local testing.